packages/jest-runner/src/runTest.ts

```diff
@@ -259,7 +259,12 @@ async function runTestInternal(
       result.numPendingTests +
       result.numTodoTests;
 
-    result.perfStats = {end: Date.now(), start};
+    const end = Date.now();
+    result.perfStats = {
```

Comment:
should we stick `runtime` or something on here as well so we don't have to do `end-start` in multiple places?

Reply:
@Draco-Umbratilis just this comment and we should be good to go

Reply:
I found one occurrence in jest-test-sequencer and made the change also there. Reflected in tests and changelog, I hope it is ok.

Reply:
Yeah, I like it!